spec/liquid_tags/unified_embed/registry_spec.rb

```diff
@@ -3,12 +3,18 @@
 RSpec.describe UnifiedEmbed::Registry do
```

Comment:
Note: I alphabetized the specs in this file appropriately, so some unrelated specs may appear to have been written over, but they were not. 😅